### DISCUSSION TITLE
Emojis via :<name>: not rendering/being displayed

### DISCUSSION THREAD

**Discussion by RihanArfan** [community/community#77606] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Since earlier today, emojis used anywhere (commit messages, pull request titles, even comments like this) no longer render if they're written as :\<name\>:

Commit message: `refactor(types): :truck: separate types to dedicated package`
Previously rendered: `refactor(types) 🚚 separate types to dedicated package`

Here's a [list of emojis](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md) names, which used to work. ([API](https://docs.github.com/en/rest/emojis/emojis?apiVersion=2022-11-28#get-emojis))

### Preview
It's a bit confusing in comments like these - see screenshots of markdown vs preview

markdown:
![image](https://github.com/community/community/assets/20425781/526bc85d-88ef-4dc8-8d32-857c66839001)

preview:
refactor(types): :truck: separate types to dedicated package
:truck: separate types to dedicated package
:truck:

### Images
Currently
![image](https://github.com/community/community/assets/20425781/abefc112-c489-4eb5-b22f-381099d11635)

Yesterday, and on presumably cached views
![image](https://github.com/community/community/assets/20425781/51f0f658-6c1f-4aff-b184-712bee4c96ec)

**Comment by cthier-eb** [comment]
I have the same issue using chrome 119.0.6045.123.

**Comment by haxcod** [comment]
respone #1

**Reply by RihanArfan** [reply]
I'm not quite sure what you mean by this, sorry.

I'm guessing you mean this community discussions is for features only, however, GitHub's topic areas include bug, and they have categories for each section of GitHub too.
![image](https://github.com/community/community/assets/20425781/340fc6df-e134-4489-9462-aca120912805)

**Comment by alvarovalverde03** [comment]
Same problem. I came to report this bug.
Since a few days ago markdown emojis markup are not working in commit messages and stay with MD plain text (:bug:).

I hope it will be solved soon

**Comment by rolfbjarne** [comment]
This happened somewhere between these two moments in time:

```
Date: Fri, 01 Dec 2023 07:01:19 -0800 [works]
Date: Fri, 01 Dec 2023 09:05:13 -0800 [fails]
```

**Comment by samcarton** [comment]
I have noticed the same thing happening in action job summaries

**Comment by GabrielLins64** [comment]
Hi, I just pushed a commit message **starting** with the emoji code `:sparkles:` and it did showed up.

![image](https://github.com/community/community/assets/42005625/0e7bfcd9-97b6-44e1-937a-8b226f9f8ad5)

 The issue seems to raise only when adding "_something:_ " **before** the emoji. But anyway, this wasn't the default behavior and breakes old commit messages, so I hope they fix this...

**Reply by RihanArfan** [reply]
Yeah, I guess some RegEx was changed somewhere to cause this 😆

**Comment by exaexa** [comment]
+1 here. Only some emoji are affected, like :x: doesn't render for me in labels, and :sweat_smile: doesn't seem to render in text.

**Reply by exaexa** [reply]
(and here it seems to work all ok... :zany_face: )

**Reply by RihanArfan** [reply]
When typing them in the GitHub UI, clicking/press enter will replace with a unicode emoji.

![image](https://github.com/community/community/assets/20425781/2799ff8c-a88f-4ac8-8d9d-0a37413ce53f)
![image](https://github.com/community/community/assets/20425781/109b399d-b858-4e74-8929-1580c0e95fca)

**Comment by suzuki-shunsuke** [comment]
I'm facing the similar issue. https://github.com/suzuki-shunsuke/tfcmt/issues/1061
In my case, it seems the issue occurs if lines start and end with emojis.

:warning: hello :warning:
:warning: hello :cry:
:warning: hello :warning: :warning:
:warning: hello
:warning: :warning: hello
:warning: hello :warning: world
hello :warning:
Yo :warning: hello :warning: world

**Comment by GabrielLins64** [comment, marked as answer]
@RihanArfan I believe they just fixed it:
![image](https://github.com/community/community/assets/42005625/3e87466f-7772-498b-95b7-aeab76ca9220)
